packages/shared/src/components/checklist/NewSquadAndSiteMemberChecklistCard.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { ReactElement } from 'react';
+import { Squad } from '../../graphql/sources';
+import { ChecklistCard } from './ChecklistCard';
+import { InstallExtensionChecklistStep } from './InstallExtensionChecklistStep';
+import { ChecklistStep } from './ChecklistStep';
+import { Button } from '../buttons/Button';
+import { ChecklistStepProps } from '../../lib/checklist';
+import BellIcon from '../icons/Bell';
+import { FlexRow } from '../utilities';
+
+const NewSquadAndSiteMemberChecklistCard = ({
```

Comment:
I'm not 100% sure this format is correct, in the latest spec we decided that steps are fully optional based on role + status.
So someone could be a multiple or roles rendering different checklist.
Did you take that into account?

Reply:
Yeah, steps creation is completely optional and can be what ever, this is just an example of one implementation.

Reply:
I'll refactor to show concrete example for the spec.

Reply:
@rebelchris see `SquadChecklistCard`